### ISSUE TITLE
Advice on fine tuning to improve accuracy on digits and special characters

### ISSUE THREAD

**Issue by itsainii** [clovaai/deep-text-recognition-benchmark#432]
I'm trying to fine tune EasyOCR to improve accuracy with digits and special characters. I found this dataset on [Kaggle](https://www.kaggle.com/datasets/loayhassanali/characters-and-digits-for-classification?resource=download) and applied the method outlined in this [tutorial](https://freedium.cfd/https://pub.towardsai.net/how-to-fine-tune-easyocr-to-achieve-better-ocr-performance-1540f5076428).

I tested with 40 images just for testing, and here are the results I obtained:

exp_name: TPS-ResNet-BiLSTM-Attn-Seed1111
train_data: lmbd_archive2/
valid_data: lmbd_archive2/
manualSeed: 1111
workers: 0
batch_size: 1
num_iter: 10
valInterval: 5
saved_model: TPS-ResNet-BiLSTM-Attn.pth
FT: False
adam: False
lr: 1
beta1: 0.9
rho: 0.95
eps: 1e-08
grad_clip: 5
baiduCTC: False
select_data: ['/']
batch_ratio: ['0.5']
total_data_usage_ratio: 1.0
batch_max_length: 80
imgH: 32
imgW: 100
rgb: False
character: 0123456789abcdefghijklmnopqrstuvwxyz
sensitive: False
PAD: False
data_filtering_off: True
Transformation: TPS
FeatureExtraction: ResNet
SequenceModeling: BiLSTM
Prediction: Attn
num_fiducial: 20
input_channel: 1
output_channel: 512
hidden_size: 256
num_gpu: 1
num_class: 38
---------------------------------------

[1/10] Train loss: 8.93992, Valid loss: 7.16805, Elapsed_time: 1.62669
Current_accuracy : 0.000, Current_norm_ED  : 0.00
Best_accuracy    : 0.000, Best_norm_ED     : 0.00
--------------------------------------------------------------------------------
Ground Truth              | Prediction                | Confidence Score & T/F
--------------------------------------------------------------------------------
                          | sichasism                 | 0.0064	False
--------------------------------------------------------------------------------
[5/10] Train loss: 4.55050, Valid loss: 1.03087, Elapsed_time: 6.90535
Current_accuracy : 65.854, Current_norm_ED  : 0.00
Best_accuracy    : 65.854, Best_norm_ED     : 0.00
--------------------------------------------------------------------------------
Ground Truth              | Prediction                | Confidence Score & T/F
--------------------------------------------------------------------------------
                          | iisile                    | 0.0647	False
--------------------------------------------------------------------------------
[10/10] Train loss: 0.01275, Valid loss: 0.22137, Elapsed_time: 10.19817
Current_accuracy : 95.122, Current_norm_ED  : 0.00
Best_accuracy    : 95.122, Best_norm_ED     : 0.00
--------------------------------------------------------------------------------
Ground Truth              | Prediction                | Confidence Score & T/F
--------------------------------------------------------------------------------
                          |                           | 0.0000	True
--------------------------------------------------------------------------------
end the training

Any advice or suggestions on where I should go from here?
